client/src/components/RelatedItems/style.css

```diff
@@ -104,6 +104,15 @@
   user-select: none;
 }
 
+.ri-star {
+  position: absolute;
+  top: 1px;
+  right: 50px;
+  font-size: 3rem;
+  z-index: 10;
+  cursor: pointer;
+}
+
 .comp-additem {
   position: absolute;
```

Comment:
I think this line might be the one that is causing the + icon to draw over the Q&A widget